src/crypto/ctaes/ctaes.c

```diff
@@ -0,0 +1,556 @@
+ /*********************************************************************
```

Comment:
I haven't checked this implementation.

Reply:
It is a direct import (via `git subtree`) of https://github.com/bitcoin-core/ctaes

Reply:
The implementation itself was reviewed by Ayo Akinyele (small world!) in 2016: https://github.com/bitcoin/bitcoin/pull/7689#issuecomment-219830217

Reply:
I also did not review it but I'm satisfied the tests (and that review!) are good enough.